apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-full-view-presenter.tsx

```diff
@@ -0,0 +1,32 @@
+import { IMultilingualText } from '@coscrad/api-interfaces';
+import { Card, CardContent, CardHeader } from '@mui/material';
+import { ReactNode } from 'react';
+import { MultilingualTextPresenter } from '../multilingual-text-presenter';
+
+export interface ResourceDetailFullViewPresenterProps {
+    id: string;
+    imageUrl?: string;
+    videoUrl?: string;
+    audioUrl?: string;
+    name: IMultilingualText;
+    children: ReactNode;
+}
+
+export const ResourceDetailFullViewPresenter = ({
+    id,
+    name,
+    audioUrl,
+    children,
+}: ResourceDetailFullViewPresenterProps): JSX.Element => (
+    <div data-testid={id}>
+        <Card>
+            <CardHeader>
+                <MultilingualTextPresenter text={name} />
+            </CardHeader>
+            {/* <CardMedia>
```

Comment:
Remove Log

Reply:
The comment is a placeholder. I probably won't be the one to put the finishing touches on the presentation, but I want to make sure we are using the `CardMedia` section so that we are using the card idiomatically.

Reply:
I'll add a todo comment to support this.

Reply:
done